modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.es.js

```diff
@@ -83,7 +83,7 @@ function toArray(value = '') {
 		newValue = [newValue];
 	}
 
-	return newValue;
+	return newValue.map((value) => value.toString());
```

Comment:
Hi @matyaswollner! Please, we'll need to use the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here, because the tests identified a place that the `newValue` array receives **_undefined_** values.

Thanks!

```suggestion
	return newValue.map((value) => value?.toString());
```